packages/orchestration/src/cosmos-api.ts

```diff
@@ -309,6 +310,16 @@ export interface LiquidStakingMethods {
   liquidStake: (amount: AmountArg) => Promise<void>;
 }
 
+export interface NobleMethods {
```

Comment:
add a docstring, please?

re **Documentation Considerations**, "No impact." doesn't seem right. This seems like a significant enhancement that merits reference docs (i.e. JSDoc), at least.